packages/terra-base/src/Base.jsx

```diff
@@ -23,6 +23,7 @@ const propTypes = {
       return new Error(`Missing locale prop for ${propName} in ${componentName} props`);
     }
   },
+  strictMode: PropTypes.bool,
```

Comment:
want to add a descriptive comment? something like `activates additional checks and warnings for descendants`

Reply:
Updated: https://github.com/cerner/terra-core/pull/1890/commits/05a761b6f58b7a4e600044b670c7a6071f00aa48